conf/webknossos.latest.routes

```diff
@@ -167,7 +167,6 @@ GET           /annotations/:typ/:id/loggedTime
 
 GET           /annotations/source/:accessTokenOrId                                              controllers.AnnotationPrivateLinkController.annotationSource(accessTokenOrId: String, userToken: Option[String])
 
-GET           /annotations/shared                                                               controllers.AnnotationController.sharedAnnotations()
```

Comment:
I guess you talked to @philippotto about whether this was used somewhere?

Reply:
Yes (see above)